README.md

```diff
@@ -14,10 +14,6 @@
 
 [Live Demo Link](https://athaliah8.github.io/Mobile_Portfolio/)
 
-
-
-
-
 ### Prerequisites
 A code complier and vscode would work great.
 
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you use the updated Microverse README **[template](https://github.com/microverseinc/readme-template)**.